ftest.sh

```diff
@@ -71,13 +70,8 @@ pre_clean () {
     i=5
     while [ $i -gt 0 ]; do
         if clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh \
-              -S -w "$(IFS=','; echo "${nodes[*]}")" "set -x -e
-              mapfile -t ftest_mounts < <(grep 'added by ftest.sh' /etc/fstab)
-              for n_mnt in \"\${ftest_mounts[@]}\"; do
-                  mpnt=(\${n_mnt})
-                  sudo umount \${mpnt[1]}
-              done
-              sudo sed -i -e \"/added by ftest.sh/d\" /etc/fstab"; then
+              -S -w "$(IFS=','; echo "${nodes[*]}")"                       \
+              "$(sed -e '1,/^$/d' "$PREFIX"/lib/daos/TESTING/ftest/scripts/pre_clean_nodes.sh)"; then
```

Comment:
(style)  line over 80 characters

Reply:
Can't be reasonably broken due to long strings and indentation.